.config/dictionaries/terminology.dic

```diff
@@ -16,6 +16,8 @@ suri # Substrate Uri
 parathread
 parathreads
 CurrencyId
+extrinisic
+extrinisics
```

Comment:
Extrinsics is already in the dictionary and these contain typos.
```suggestion
```

Reply:
fixed